tendrl_unit_test_setup.yml

```diff
@@ -28,3 +29,4 @@
       yum:
         name: 'tendrl*'
         state: latest
+        exclude: 'tendrl-node-selinux'
```

Comment:
Are you sure that this is correct? Is there any issue related to removal of this package?

Reply:
Ok, my mistake.